addons/namingng.py

```diff
@@ -266,8 +266,10 @@ def report_pending_ifndef(directive,column):
 def process(dumpfiles, configfile):
     conf = loadConfig(configfile)
 
+    # pylint: disable-next=no-member - TODO: fix this
```

Comment:
@thingsconnected I do not understand why this is not triggered for the other cases where this class is used.